src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -32,6 +32,9 @@
 
     <BuildArgs Condition="'$(UseSystemLibs)' != ''">$(BuildArgs) /p:UseSystemLibs=$(UseSystemLibs)</BuildArgs>
 
+    <!-- When we're actually doing signing and the ESRP tool is available, forward down the path to the repo builds. -->
+    <BuildArgs Condition="'$(Sign)' == 'true' and '$(ForceDryRunSigning)' == '' and '$(DotNetEsrpToolPath)' != ''">$(BuildArgs) /p:DotNetEsrpToolPath=$(DotNetEsrpToolPath)</BuildArgs>
```

Comment:
Should we pass that in as a global property for all repos?

Reply:
I would rather only pass this down for repositories that must use ESRP directly. We want to limit direct ESRP usage to only necessary scenarios.